datafusion/core/tests/parquet/filter_pushdown.rs

```diff
@@ -636,6 +636,27 @@ async fn predicate_cache_pushdown_default() -> datafusion_common::Result<()> {
     config.options_mut().execution.parquet.pushdown_filters = true;
     let ctx = SessionContext::new_with_config(config);
     // The cache is on by default, and used when filter pushdown is enabled
+    PredicateCacheTest {
+        expected_inner_records: 8,
+        expected_records: 7, // reads more than necessary from the cache as then another bitmap is applied
```

Comment:
this behavior changed due to adaptive filtering. I added a new test that turns off adaptive filtering to show doing so restores the old behavior